data/na/swagger/1-0-5.yaml

```diff
@@ -2010,6 +2037,50 @@ definitions:
       eci:
         type: integer
         description: 'The electronic commerce indicator (ECI) used for processing transactions with 3D Secure.'
+  BrowserData:
+    description: |
+      Data collected about the cardholders browser, required for 3DS 2.0 transactions, optional for others.
+    required:
+      - acceiptHeader
+      - javaEnabled
+      - language
+      - screenHeight
+      - screenWidth
+      - timeZone
+      - userAgent
+      - JavascriptEnabled
+    properties:
+      acceptHeader:
+        type: string
+        maxLength: 2048
+        description: 'Accept Header returned by the browser.'
+      javaEnabled:
+        type: boolean
+        description: 'True if the browser supports Java.'
+      language:
+        type: string
+        maxLength: 8
+        description: 'Language currently set in the browser.'
+      colorDepth:
+        type: integer
+        description: 'Color bit-depth of the screen.'
+      screenHeight:
+        type: integer
+        description: 'Height, in pixels, of the screen.'
+      screenWidth:
+        type: integer
+        description: 'Width, in pixels, of the screen.'
+      timeZone:
+        type: integer
+        description: 'Timezone returned by the browser.'
+      userAgent:
+        type: string
+        maxLength: 2048
+        description: 'User-Agent header returned by the browser.'
```

Comment:
This should mention that it comes in the client HTTP request, as it cannot be fetched in javascript like the other values